src/components/cards/AvatarList.vue

```diff
@@ -70,14 +70,14 @@
 </template>
 
 <script>
-import { Avatar, PopoverMenu, Tooltip } from '@nextcloud/vue'
+import { NcAvatar, NcPopoverMenu, Tooltip } from '@nextcloud/vue'
```

Comment:
Does Tooltip stay without prefix?

Reply:
Yes, only components require the two word naming enforced by eslint.